### PR TITLE
Mirror: Added cheap sunglasses into ClothesMate

### PR DESCRIPTION
**Pull request by SimpleStation14**
###### `d370f16e04fc12c8fa2b89c41c3fce12225334b4`

PR opened by <a href="https://github.com/shampunj"> shampunj</a> at 2024-02-22 21:58:13 UTC

---

PR changed 3 files with 15 additions and 7 deletions.

The PR had the following labels:
- No C#
- Status: Needs Review
- Merge Conflict

---

<details open="true"><summary><h1>Original Body</h1></summary>

> 
> 
> 
> ## About the PR
> 
> Added cheap sunglasses that wasnt protect against flashing into ClothesMate
> 
> ## Why / Balance
> 
> To replace removed regular sunglasses in ClothesMate https://github.com/space-wizards/space-station-14/pull/25221
> ## Technical details
> 
> n/a
> ## Media
> 
> 
> - [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
> Screenshot_53
> Screenshot_54
> 
> 
> ## Breaking changes
> 
> 
> **Changelog**
> 
> 
> 
> - add: Added cheap sunglasses that wasnt protect against flashing into ClothesMate
> 

</details>

### CODE REVIEW

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
needs price changes

**Review by VMSolidus — APPROVED** [review]
Price changes are done, and this is next on the Pre-requisites list for DeltaV PRs

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml [hunk 1]
```diff
@@ -33,8 +33,8 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 7250 #DeltaV
-  category: Service
+  cost: 7245
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]: okay i don't wanna be *that* guy but this should be 7250 or something 😢

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml (hunk 1 above)
Comment (VMSolidus) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]: Infinite money exploit obtained, repeatedly buying and reselling clothesmate stocks for 5 spesos each lmao.

Comment (DangerRevolution) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml]: buy and sell prices are the same? that's insane.

Comment (DangerRevolution) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml]: put it back on 7245

Comment (VMSolidus) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]: Lol. Yea it's to prevent people from doing cargo exploits.

**Review by DangerRevolution — APPROVED** [review]
There *really* should be different buy / sell prices. So you can buy something for 10K and sell for 0. As an example.

**Comment by VMSolidus** [comment]
I mean you can, it's just that we can't actually set the sell price of crates directly because it's summed by all the contents of the crates. The system only cares that sell price must be equal to or less than buy price, but we can only set the buy price directly.

**Comment by AeraAuling** [comment]
> I mean you can, it's just that we can't actually set the sell price of crates directly because it's summed by all the contents of the crates. The system only cares that sell price must be equal to or less than buy price, but we can only set the buy price directly.

i'd suggest asking frontier about stuff like this, they have stuff like the cargo crates that don't adhere to this system by selling for more then you buy them for. dunno how they do it, but they might have worked out a way to set sell prices as well

**Comment by DEATHB4DEFEAT** [comment]
> they have stuff like the cargo crates that don't adhere to this system by selling for more then you buy them for

That's not good.

**Comment by VMSolidus** [comment]
> > they have stuff like the cargo crates that don't adhere to this system by selling for more then you buy them for
> 
> That's not good.

For context the crates in question can only be sold at a profit at an entirely different station than the one you bought them at. You get money in exchange for transporting them to another station.

**Pull request merged**